forecasttools/daily_to_epiweekly.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+
+import polars as pl
+
+
+# NOTE: this needs to change to ACTUAL epiweeks
```

Comment:
Still to do?

Reply:
Yes. I was going to ask DHM if there was `lubridate` `epiweek` equivalent or pre-checked `polars` code he has. I've done this before but wanted to wait for a standard procedure.

Reply:
I'd use https://epiweeks.readthedocs.io/en/stable/